public/userview.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!-- Referenced https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL -->
+    <script>
+        function showImg() {
+            const preview = document.querySelector('img');
+            const file = document.querySelector('input[type=file]').files[0];
+            const reader = new FileReader();
+            reader.addEventListener("load", function () {
+                // convert image file to base64 string
+                preview.src = reader.result;
+            }, false);
+
+            if (file) {
+                reader.readAsDataURL(file);
+            }
+        }
+    </script>
+
+    <!-- Referenced https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch -->
+    <script>
+        function upload() {
+            document.getElementById("form1").addEventListener("click", function(event){
+                event.preventDefault()
+            });
+
+            const formData = new FormData();
```

Comment:
You can do `new FormData(document.getElementById("form1"))`

Reply:
done